var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -25,20 +26,40 @@ class Draco(CMakePackage):
     version('6_20_1', sha256='b1c51000c9557e0818014713fce70d681869c50ed9c4548dcfb2e9219c354ebe')
     version('6_20_0', sha256='a6e3142c1c90b09c4ff8057bfee974369b815122b01d1f7b57888dcb9b1128f6')
 
-    variant('lapack',   default=False, description='Enable LAPACK Wrapper')
+    variant('build_type', default='Release', description='CMake build type',
+            values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
     variant('eospac',   default=False, description='Enable EOSPAC Support')
+    variant('lapack',   default=False, description='Enable LAPACK Wrapper')
     variant('parmetis', default=False, description='Enable Parmetis Support')
     variant('qt',       default=False, description='Enable Qt Support')
     variant('superlu_dist', default=False, description='Enable SuperLU-DIST Support')
+    variant("test",     default=False, description="Build Draco regression tests")
```

Comment:
@adamjstewart is there a way to enable with automatically with `--run-tests`?

Reply:
@junghans Is there a good `package.py` to use as an example.  I would like to enable the option of running our tests.

Reply:
You can do `spack install --run-tests draco`.

Reply:
Actually `spack install --test=root draco` these days.

Reply:
Inside the package, you can check the value of `self.run_tests` (assuming you have access to self). Inside of a dependency, you can use `depends_on('...', type='test')` for a test-only dependency. And yes, you can enable tests on the command line with `--test=root` or `--test=all`. We try to avoid test variants because they result in duplicate installations if you install `+test` and `~test` even though the installation is the same.

Reply:
For style sake, can you stay consistent on the quotes?

Reply:
I removed the `test` variant (which removes the quote inconsistency) and replaced it with support for `--test=root`.